vunit/com/codec_vhdl_package.py

```diff
@@ -148,7 +148,7 @@ def _create_enumeration_of_all_msg_types(self):
 
         msg_type_enumeration_types = []
         for record in self.record_types:
-            if record.elements[0].identifier_list[0] == "msg_type":
+            if record.elements[0].identifier_List[0] == "msg_type":
```

Comment:
Is this really correct? Seems a bit like a search-and-replace-error. `list[` -> `List[`

Reply:
Thank you good catch. Thats exactly what happened.